src/rite-app.html

```diff
@@ -49,9 +48,6 @@
 /*global Prism*/
 // Match all lines that are not insertions or deletions
 Prism.languages.diff.other = /.+/m;
-// Additions and deletions can also be an empty line
```

Comment:
Is this intended that empty lines are not coloured anymore?

![image](https://cloud.githubusercontent.com/assets/5946438/15888936/516d5bc6-2d69-11e6-9302-7bfe086e0c5a.png)